src/dysh/spectra/spectrum.py

```diff
@@ -1400,3 +1509,66 @@ def spectrum_reader_gbtidl(fileobj, **kwargs):
     # registry.register_writer("ipac", Spectrum, ascii_spectrum_reader_ipac)
     # registry.register_writer("votable", Spectrum, spectrum_reader_votable)
     # registry.register_writer("mrt", Spectrum, spectrum_reader_mrt)
+
+
+def average_spectra(spectra, equal_weights=False):
```

Comment:
This should be moved to core.py and eventually replace use of  average() in time averaging.

Reply:
I couldn't figure out how to check for `Spectrum` instances from core due to a circular import issue.

Reply:
Also, `core.average` and `spectrum.average_spectra` serve two different use cases. The former is for averaging inside a `Scan`, the later is for averaging `Spectrum`. To replace one with the other we would need to turn every integration into a `Spectrum`, which would be slower than needed.